pact/coin-contract/v5/coin-v5.pact

```diff
@@ -545,6 +547,7 @@
 
         (validate-account sender)
         (validate-account receiver)
+        (enforce (!= sender receiver) "sender and receiver should differ")
```

Comment:
aha but they _can_ be the same! Just, they should exist on different chains, which we have no means of expressing in the current system.

Reply:
this is enforced by the `crosschain-transfer` of the `fungible-v2` interface, see https://github.com/kadena-io/chainweb-node/blob/master/pact/coin-contract/v2/fungible-v2.pact#L94

Reply:
`fungible-v2` has an invalid property then :disappointed: